src/test/munit/sftp-read-test-case.xml

```diff
@@ -21,6 +21,18 @@
         </sftp:connection>
     </sftp:config>
 
+    <sftp:config name="strict-KEX-enabled-config">
```

Comment:
This is not being use this should be in the parameterization side

Reply:
Done, thanks